docs/application/dotnet/guides/text-input/voice-control-manager.md

```diff
@@ -0,0 +1,1096 @@
+
+            /// Unset the callback
+            VoiceControlManagerClient.ConversationRequested -= conversationRequested;
+        }
+        catch (Exception e)
+        {
+            /// Error handling
+        }
+    }
+    ```
+
+-   Set the private data updated callback invoked when the voice control engine sets private data to the voice control manager:
+    ```csharp
+    /// callback
+    void privateDataUpdated(object sender, PrivateDataUpdatedEventArgs e)
+    {
+        /// Your code
+    }
+
+    void SetUnsetPrivateDataUpdatedCb()
+    {
+        try
+        {
+            /// Set the callback
+            VoiceControlManagerClient.PrivateDataUpdated += privateDataUpdated;
+
+            /// Unset the callback
+            VoiceControlManagerClient.PrivateDataUpdated -= privateDataUpdated;
+        }
+        catch (Exception e)
+        {
+            /// Error handling
+        }
+    }
+    ```
+
+
+-   Set the specific engine result callback invoked when voice control engine sends additional information about recognition result not the registered commands:
+
+    ```csharp
+    /// callback
+    void specificEngineResult(object sender, SpecificEngineResultEventArgs e)
+    {
+        /// Your code
+    }
+
+    void SetUnsetSpecificEngineResultCb()
+    {
+        try
+        {
+            /// Set the callback
+            VoiceControlManagerClient.SpecificEngineResult += specificEngineResult;
+
+            /// Unset the callback
+            VoiceControlManagerClient.SpecificEngineResult -= specificEngineResult;
+        }
+        catch (Exception e)
+        {
+            /// Error handling
+        }
+    }
+    ```
+
+
+-   Set the pre recognition result updated callback invoked when voice control engine sets the pre recognition results (partial ASR) to voice control manager.
+
+    You can get the results of pre result event type and the pre recognition results, the partial ASR.
+
+    ```csharp
+    /// callback
+    void preRecognitionResultUpdated(object sender, PreRecognitionResultUpdatedEventArgs e)
+    {
+        /// Your code
+    }
+
+    void SetUnsetPreRecognitionResultUpdatedCb()
+    {
+        try
+        {
+            /// Set the callback
+            VoiceControlManagerClient.PreRecognitionResultUpdated += preRecognitionResultUpdated;
+
+            /// Unset the callback
+            VoiceControlManagerClient.PreRecognitionResultUpdated -= preRecognitionResultUpdated;
+        }
+        catch (Exception e)
+        {
+            /// Error handling
+        }
+    }
+    ```
+
+
+-   Set the all recognition result received callback invoked when voice control engine sends the all recognition result to voice control manager.
+
+    In the callback, the recognized result, recognized text, and engine message results are included and recognized result can include more than two voice commands if two voice commands those have same command string are registered by each voice control clients.
```

Comment:
Suggesting rephrase as:

In the callback, the recognized result, recognized text, and engine message results are included and recognized result can include more than two voice commands. If two voice commands those have same command string, are registered by each voice control clients.

Please verify.

Reply:
@mijin85cho,

I discussed with @rishu-samsung and suggest the following:

In the callback, the recognized result, recognized text, and engine message results are included. Recognized result can include more than two voice commands, if the two voice commands have the same command string and are registered by each voice control clients.

Please verify and let us know your views.

Reply:
Thanks for the discussion, @surbhig12.
I think your suggestion is good. I hope @sungraejo to check the suggestion and apply it if there is no issue.

Reply:
@surbhig12 Thank you your suggestion.
@mijin85cho I will apply it.